core/services/metrics.go

```diff
@@ -0,0 +1,60 @@
+package services
+
+/////////// ::::: TODO DAVE ::::
+///
+///////////     Split this into endpoint and service?
+///
+///////////     THEN decide if startupOptions or other schema package has the wrong reference toLocalAiMetrics - it seems more like the service itself [perhaps rename to LocalAIMetricsService]
+
+import (
+	"context"
+
+	"github.com/rs/zerolog/log"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/prometheus"
+	"go.opentelemetry.io/otel/metric"
+	metricApi "go.opentelemetry.io/otel/sdk/metric"
+)
+
+type LocalAIMetricsService struct {
+	Meter         metric.Meter
+	ApiTimeMetric metric.Float64Histogram
+}
+
+func (m *LocalAIMetricsService) ObserveAPICall(method string, path string, duration float64) {
+	opts := metric.WithAttributes(
+		attribute.String("method", method),
+		attribute.String("path", path),
+	)
+	m.ApiTimeMetric.Record(context.Background(), duration, opts)
+}
+
+// setupOTelSDK bootstraps the OpenTelemetry pipeline.
+// If it does not return an error, make sure to call shutdown for proper cleanup.
+func NewLocalAiMetricsService() (*LocalAIMetricsService, error) {
```

Comment:
```suggestion
func NewLocalAIMetricsService() (*LocalAIMetricsService, error) {
```

For consistency (also we return a `LocalAIMetricsService` here)